aws-aurora-postgresql.yml

```diff
@@ -105,12 +103,25 @@ provision:
     constraints:
       minimum: 1
       maximum: 35
-  - <<: *nullable_string
-    field_name: preferred_backup_window
+  - field_name: preferred_backup_window
+    type: string
+    default: null
+    nullable: true
     details: |
       The daily time range in UTC during which automated backups are created, e.g.: "09:46-10:16".
       Must not overlap with the maintenance window. If not set, uses the default for the region
       (see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Backups.html#Aurora.Managing.Backups.BackupWindow)
+  - field_name: require_ssl
+    type: boolean
+    details: Require that connections use SSL. Note that if "db_cluster_parameter_group_name" is specified then the "require_ssl" parameter will not take effect.
+    default: true
+  - field_name: db_cluster_parameter_group_name
```

Comment:
**question**: What is the advantage of empty string vs. null here?

Reply:
I haven't thought about the advantages and disadvantages. I checked the implementation in the AWS provider and see that a string is expected and set if the value is not the zero value. Do you want me to change it?